packages/usa-icon/src/usa-icons.config.js

```diff
@@ -0,0 +1,251 @@
+const config = {
+  material: [
```

Comment:
If I have a single item entry in any of the `config` items I get an error. Below is an example of a single entry in material key.

```js
const config = {
  material: ["wash"],
},
materialDeprecated: […defaults],
uswds: […defaults]
```

```bash
[13:18:06] Error: File not found with singular glob: /home/jmejia/web/uswds/packages/usa-icon/src/img/material-icons-deprecated/{eco}.svg (if this was purposeful, use `allowEmpty` option)
    at Glob.<anonymous> (/home/jmejia/web/uswds/node_modules/glob-stream/readable.js:84:17)
    at Object.onceWrapper (node:events:510:26)
    at Glob.emit (node:events:390:28)
    at Glob.emit (node:domain:475:12)
    at Glob._finish (/home/jmejia/web/uswds/node_modules/glob/glob.js:194:8)
    at done (/home/jmejia/web/uswds/node_modules/glob/glob.js:179:14)
    at Glob._processSimple2 (/home/jmejia/web/uswds/node_modules/glob/glob.js:688:12)
    at /home/jmejia/web/uswds/node_modules/glob/glob.js:676:10
    at Glob._stat2 (/home/jmejia/web/uswds/node_modules/glob/glob.js:772:12)
    at lstatcb_ (/home/jmejia/web/uswds/node_modules/glob/glob.js:764:12)
[13:18:06] 'buildSpriteStandalone' errored after 1.76 s
```

Reply:
Closing in favor of https://github.com/uswds/uswds/issues/4822